WebRoot/public/login.jsp

```diff
@@ -93,7 +93,7 @@
 		</c:when>
 		<c:when test="${param.loginOp eq 'relogin' and not empty param.loginErrorCode}">
 			<zm:logout/>
-			<c:set var="errorCode" value="${param.loginErrorCode}"/>
+			<c:set var="errorCode" value="${fn:escapeXml(param.loginErrorCode)}"/>
```

Comment:
check what is difference between fn:escapeXml and zm:cook, I prefer to use zm:cook because escapeXml shows that we are escaping xml but it is not the case

Reply:
Fixed.